app/models/computer_game.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class ComputerGame < ApplicationRecord
+  include ActiveModel::Validations
+
+  belongs_to :user
+  has_one :computer, dependent: :destroy
+  has_many :games, dependent: :destroy
+
+  validates_with ComputerGameValidator
+  validate :computer_is_used, on: :create
```

Comment:
this validation is separate from ComputerGameValidator because it's the only one that we want to have on create action